modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-network-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Creating an additional OVN-Kubernetes `br-ex` bridge network
+
+You can create an additional `br-ex` bridge network, `br-ex1`, that has a different configuration to the default `br-ex` bridge. You must define the additional bridge in the target interface definition of a machine config manifest file.  
+
+After you create and edit the manifest file, the Machine Config Operator completes tasks in the following order:
+
+* Drains nodes in singular order based on the selected machine config pool.
+* Injects Ignition configuration files into each node, so that each node receives the additional `br-ex` bridge network configuration.
+* Restarts the node with the modified override file.
+* Executes the `configure-ovs.sh` shell script that references the new interface definition.
+* Adds `br-ex` and `br-ex1` to the host node.
+* Uncordons the nodes. After all the nodes return to the `Ready` state, the Operator applies the `k8s.ovn.org/l3-gateway-config` annotation to each node.
+
+For more information about useful situations for the additional `br-ex` bridge and a situation that always requires the default `br-ex` bridge, see "Configuration for a localnet topology".
+
+.Procedure
+
+. Optional: Create an interface connection that your secondary interface, `br-ex1`, can use by completing the following steps. The steps use example configurations to show the use of two interfaces instead of a machine config manifest file to define a secondary bond interface for your cluster. 
```

Comment:
Is this really optional. In other words it could be skipped and a user could continue from "Prepare the environment variables"? This assumes that they already have existing interface definition files?

Reply:
I could add information about skipping this step if x and y already exists.